### PR TITLE
Base 268 3

### PR DESCRIPTION
**Pull request by CeltonMcGrath**

### CODE REVIEW

File: src/tactic/ui/tools/repo_browser_wdg.py [hunk 1]
```diff
@@ -685,34 +728,40 @@ def get_relative_paths(my, base_dir):
 
         # add dirnames if they have sobject files in them
         #if not show_no_sobject_folders:
+        new_sub_paths = []
         if os.path.exists(base_dir) and os.path.isdir(base_dir):
-            dirnames = os.listdir(base_dir)
+            dirnames = os.listdir(unicode(base_dir))
```
Comment (CeltonMcGrath) [inline src/tactic/ui/tools/repo_browser_wdg.py]: if base_Dir already unicode, do not use unicode.

File: src/tactic/ui/tools/repo_browser_wdg.py [hunk 2]
```diff
@@ -501,14 +512,53 @@ def init(my):
         super(RepoBrowserDirListWdg, my).init()
 
 
+    def _clean_folder_state(my, folder_state, key):
+        # Get and clean folder_states
```
Comment (oboreo) [inline src/tactic/ui/tools/repo_browser_wdg.py]: ''' get and clean '''

**Comment by oboreo** [comment]
1. chagne to this in get_folder_state() 
var items = folder_state ? folder_state("|") : [];
2. rename spt.repo_browser.getElement to get_element()

File: src/tactic/ui/tools/repo_browser_wdg.py [hunk 3]
```diff
@@ -32,17 +32,17 @@ class RepoBrowserWdg(BaseRefreshWdg):
 
     ARGS_KEYS = {
         "parent_mode": {
-            'description': '''The relationship between a snapshot and parent sObject and which determine what views are exposed and   
-                functionality of tools. When mode is single_file, the parent sObject is updated, renamed, deleted and moved with 
-                child snapshots. Note, the single_file search_type must have relative_dir specified.  
-                When mode is single_asset with given search_key, the directory and tools are scoped to the parent.''',
+            'description': '''Determines the sType that is shown in the directory listing and content wdg.
+                - single_search type shows snapshots of files found
```
Comment (oboreo) [inline src/tactic/ui/tools/repo_browser_wdg.py]: given search type

File: src/tactic/ui/tools/repo_browser_wdg.py [hunk 4]
```diff
@@ -233,14 +232,17 @@ def get_display(my):
         show_base_dir = my.kwargs.get("show_base_dir")
         
         # Dynamically load the directory listing
-        dynamic = True
+        dynamic = my.kwargs.get("dynamic")
+        if not dynamic:
+            dynamic = False
```
Comment (oboreo) [inline src/tactic/ui/tools/repo_browser_wdg.py]: just declare dynamic = False

**Pull request merged**